### DISCUSSION TITLE
How to use the provided JSON serializer?

### DISCUSSION THREAD

**Discussion by r4rmas** [activej/activej#272] [Q&A]
Hi @eduard-vasinskyi , I've seen some benchmarks that position Activej JSON serializer as the most fast serializer. Yet, I can't find any example using this serializer. Could you provide an example?

Note:
I found in the official site a [tutorial](https://activej.io/tutorials/todo-list-tutorial) but it uses DSL-JSON library which is an external library.

**Comment by eduard-vasinskyi** [comment]
Hi, @r4rmas

ActiveJ does not include a JSON parser/serializer. However, its technology-agnostic design allows you to use a JSON library of your choice.

For our purposes we sometimes use the DSL-JSON library as it seems to show the greatest performance as per [benchmarks](https://github.com/fabienrenaud/java-json-benchmark).

If you are referring to TechEmpower’s JSON benchmark, its task is to exercise keep-alive support, request routing, request header parsing, object instantiation, JSON serialization, response header generation, and request count throughput (according to [documentation](https://github.com/TechEmpower/FrameworkBenchmarks/wiki/Project-Information-Framework-Tests-Overview#json-serialization)). There are many more variables being measured than just a JSON serialization. Many frameworks there use external libraries for JSON serialization.
